db/Pos/DB/Class.hs

```diff
@@ -232,7 +257,7 @@ dbGetBlund x =
 -- to use real DB without IO. Finally, it has 'MonadCatch' constraints
 -- (partially for historical reasons, partially for good ones).
 type MonadRealDB m
-     = (Ether.MonadReader' NodeDBs m, MonadIO m, MonadCatch m)
+     = (Ether.MonadReader' NodeDBs m, MonadIO m, MonadBase IO m, MonadCatch m)
```

Comment:
Yes, there's no automatic instance of `MonadIO` from `MonadBase IO` (i dunno why).